.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: python
+python:
+  - "2.7"
+install:
+    # Requirements needed for Travis CI
+    - pip install coveralls
+    - pip install yapf
+    - pip install pyflakes
+    - pip install pylint==1.5.5
```

Comment:
Can we put `yapf`, `pyflakes`, and `pylint` in a file called `dev_requirements.txt` and do `pip install -r dev_requirements.txt`? Also, `scrapy` should go in a file called `requirements.txt` and that should have an explicit version too for the sake of stability.

Coveralls can stay here because we only need it on Travis builds.